e2e/keywords/deployment.resource

```diff
@@ -4,13 +4,21 @@ Documentation    Deployment Keywords
 Library    Collections
 Library    ../libs/keywords/common_keywords.py
 Library    ../libs/keywords/deployment_keywords.py
+Library    ../libs/keywords/node_keywords.py
 
 *** Keywords ***
 Create deployment ${deployment_id} with persistentvolumeclaim ${claim_id}
     ${deployment_name} =    generate_name_with_suffix    deployment    ${deployment_id}
     ${claim_name} =    generate_name_with_suffix    claim    ${claim_id}
     create_deployment    ${deployment_name}    ${claim_name}
 
+Create deployment ${deployment_id} with persistentvolumeclaim ${claim_id} with max replicaset
```

Comment:
We can simplify this by making an assumption that we have 3 worker nodes like:
```
def scale_statefulset_up(self, statefulset_name, replicaset_count=3):
```

Reply:
IMO, we should avoid making assumptions about the infrastructure, as the robot isn't confined to running solely within the pipeline; it may also execute on our preferred cluster, which could possess a different setup in terms of node count.